src/components/NotFoundPage.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+    import AppDetail from "../pages/AppDetail.svelte";
+    import AppList from "../pages/AppList.svelte";
+    import PingDetail from "../pages/PingDetail.svelte";
+    import MetricDetail from "../pages/MetricDetail.svelte";
+    import TableDetail from "../pages/TableDetail.svelte";
+
+ let app = new App({
+    target: document.body,
+    props: {
+        app: AppDetail,
+        list: AppList,
+        ping: PingDetail,
+        metric: MetricDetail,
+        table: TableDetail,
+    }
+})
```

Comment:
We shouldn't be creating a new app instance in a component, which is generally reserved for `main.js`. This is probably why you're getting strange issues.

In the actual pages, I would expect to something like the following in the main code. Of course, this can be implemented in a number of ways with similar behavior.

```js
{#await some_promise}
    ....
{:then ...}
   ...
{:error error}
  <NotFoundPage name={....}
{/awai}
```

The component defined here is included when fetching the resource fails. I'd recommend taking a look at the svelte tutorial on building components with props: https://svelte.dev/tutorial/declaring-props